.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Flag all PR configuration changes for documentation follow-up.
+/tiledb/sm/cpp_api/config.h   @TileDB-Inc/tiledb-docs
```

```suggestion
/tiledb/sm/cpp_api/config.h   @TileDB-Inc/tiledb-docs
/tiledb/api/c_api/config/config_api_external.h  @TileDB-Inc/tiledb-docs
```

@KiterLuc This is what you were talking about with how the config was split, correct? If so, we should have visibility into both of these files.

---

At the moment the C++ API file should always reflect the state of the config parameters (this is kept in-sync by hand right now)